apps/block_scout_web/lib/block_scout_web/controllers/address_transaction_controller.ex

```diff
@@ -232,6 +268,22 @@ defmodule BlockScoutWeb.AddressTransactionController do
     )
   end
 
+  def transactions_csv(conn, %{
+        "address_id" => address_hash_string,
+        "from_period" => from_period,
+        "to_period" => to_period
+      }) do
+    items_csv(
+      conn,
+      %{
+        "address_id" => address_hash_string,
+        "from_period" => from_period,
+        "to_period" => to_period
+      },
+      AddressTransactionCsvExporter
+    )
+  end
```

Comment:
Here I think we can replace two clauses with this one actually
```suggestion
  def transactions_csv(conn, params) do
    items_csv(conn, params, AddressTransactionCsvExporter)
  end
```